### PR TITLE
rmt-server-pubcloud: manual instance verification script

### PR DESCRIPTION
**Pull request by ikapelyukhin**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Unless this is a trivial change, please include a CHANGELOG entry.<br />
Run <code>osc vc</code> in the <code>package</code> directory to add one.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/114613/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-run-tests. (Merged 64facd68fb3b4d6686ad9a556156813b805c8a39 into 6effbb23e766e37ac1f25c3342b4f6ea857c85e2)

File: bin/rmt-manual-instance-verify [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+require 'logger'
+
+options = {}
+parser = OptionParser.new do |opts|
+  opts.banner = "Usage: #{$PROGRAM_NAME} [options]"
+
+  opts.on('-i', '--instance-data INSTANCE_DATA', 'Path to instance data file') do |val|
+    options[:instance_data] = val
+  end
+  opts.on('-m', '--module MODULE', 'Path to instance verification module') do |val|
```
Comment (rjschwei) [inline bin/rmt-manual-instance-verify:14]: Please add ". By default the verification module on the server will be loaded. (Optional)"

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/114674/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-run-tests. (Merged 64facd68fb3b4d6686ad9a556156813b805c8a39 into 6effbb23e766e37ac1f25c3342b4f6ea857c85e2)

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/114857/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-run-tests. (Merged 64facd68fb3b4d6686ad9a556156813b805c8a39 into 6effbb23e766e37ac1f25c3342b4f6ea857c85e2)

File: bin/rmt-manual-instance-verify (hunk 1 above)
Comment (ikapelyukhin) [reply bin/rmt-manual-instance-verify]: Done.

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/114867/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 51552e019b03e7af2269b9de0588cf58e57a1cc2 into 6effbb23e766e37ac1f25c3342b4f6ea857c85e2)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/116005/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 51552e019b03e7af2269b9de0588cf58e57a1cc2 into 82eff393eb69e82402b6eb3b85229d1ebf83b722)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/116006/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 51552e019b03e7af2269b9de0588cf58e57a1cc2 into 5e802f20086fa72726e59689767bb9ed26c485c9)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/116007/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 51552e019b03e7af2269b9de0588cf58e57a1cc2 into dcfbc8ef4081a266ac343cba1d283fb6866b4400)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/116008/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 51552e019b03e7af2269b9de0588cf58e57a1cc2 into 5816526b08f8aa7fb867e3056ceab435473bfeee)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/116009/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 51552e019b03e7af2269b9de0588cf58e57a1cc2 into f1c2c37f146e1aacf819c459840d039dc05d589b)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/116146/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 51552e019b03e7af2269b9de0588cf58e57a1cc2 into a76145dba0e5c82a8517b740cb7a62a1b0918db4)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/116147/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 51552e019b03e7af2269b9de0588cf58e57a1cc2 into 9a0d09621993f7855a89e7792a234af56571b879)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/116764/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 4c855e3c6542f9117c10a820c77ed83d6fe20262 into 9a0d09621993f7855a89e7792a234af56571b879)

**Pull request merged**